Branch-SDK/src/io/branch/referral/BranchShortLinkBuilder.java

```diff
@@ -127,7 +127,19 @@ public BranchShortLinkBuilder setType(int type) {
      * @return A {@link String} containing the resulting short URL. Null is returned in case of an error or if Branch is not initialised.
      */
     public String getShortUrl() {
-        return super.getUrl();
+        return super.getUrl(true);
+    }
+
+    /**
+     * <p>Configures and requests a short URL to be generated by the Branch servers, via a synchronous
+     * call; with a duration specified within which an app session should be matched to the link.</p>
+     *
+     * @param defaultToLongUrl A {@link Boolean} whose value should be false if a  long url is not intended to return on failure.
```

Comment:
a bit nitpicky: the argument isn't a Boolean, but a primitive boolean (lowercase b)